### PR TITLE
[Hub Generated] Review request for Microsoft.Portal to add version preview/2020-09-01-preview

### PR DESCRIPTION
**Pull request by msHich**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/msHich/azure-rest-api-specs/hich%2FMarkdownSourceUpdate...master/).

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [x] Removing API(s) in stable version
- [x] Removing properties in stable version
- [x] Removing API version(s) in stable version
- [x] Updating API in stable version with Breaking Change Validation errors
- [x] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/portal/resource-manager/Microsoft.Portal/preview/2020-09-01-preview/portal.json [hunk 1]
```diff
@@ -500,7 +500,8 @@
                       "description": "The subtitle of the markdown part."
                     },
                     "markdownSource": {
-                      "type": "object",
+                      "type": "integer",
```
Comment (majastrz) [inline specification/portal/resource-manager/Microsoft.Portal/preview/2020-09-01-preview/portal.json:503]: >integer [](start = 31, length = 7)

Narrowing a type is a breaking change. Can you explain?

**Comment by majastrz** [comment]
Please take a look at the versioning guidelines at  https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning. Modifying a preview version older than a year is not allowed and requires a version bump. Also, you are introducing a breaking change that requires breaking change approval. In the link I provided there are instructions on how to obtain the approval.

File: specification/portal/resource-manager/Microsoft.Portal/preview/2020-09-01-preview/portal.json (hunk 1 above)
Comment (majastrz) [reply specification/portal/resource-manager/Microsoft.Portal/preview/2020-09-01-preview/portal.json:503]: We discussed internally over email:
image

Based on this, I don't think the breaking change approval is necessary.

**Review by majastrz — APPROVED** [review]
Based on @msHich's statements, the API is not live yet and has no users, so this isn't a real breaking change.

**Comment by majastrz** [comment]
I think I was also thrown by the checkbox saying that it's a preview API older than a year. `2020-09-01-preview` definitely does not fall in that category,

**Review by markcowl — APPROVED** [review]

**Pull request merged**